src/evaluate/load.py

```diff
@@ -415,12 +420,14 @@ def __init__(
         download_config: Optional[DownloadConfig] = None,
         download_mode: Optional[DownloadMode] = None,
         dynamic_modules_path: Optional[str] = None,
+        module_namespace: Optional[str] = None,
     ):
```

Comment:
This is not just a `GithubMetricModuleFactory` anymore if it can also load a `Comparison` ?

Feel free to rename the class to something like `GithubEvaluationModuleFactory`, or copy-paste (or factorize) it to also have `GithubComparisonModuleFactory`.